client/modules/core/main.js

```diff
@@ -186,21 +186,33 @@ export default {
     // called out a userId is validated.
     //
     function roleCheck() {
+      // TODO: There're some outdated comments in this function. Clean them out.
+
       // permissions can be either a string or an array
       // we'll force it into an array and use that
       if (checkPermissions === undefined) {
         permissions = ["owner"];
       } else if (typeof checkPermissions === "string") {
         permissions = [checkPermissions];
       } else {
+        /* TODO: Consult and add the code below if it is thought
+        of as useful.
+        if (!Array.isArray(checkPermissions)) {
```

Comment:
While it doesn't do any harm, I _think_ we can drop this check. I say that because the final method called -[Roles.userIsInRole](http://alanning.github.io/meteor-roles/files/roles_roles_common.js.html#l254)- can handle both a string and an array

Reply:
Ok. Will do that.